src/main/java/com/couchbase/lite/Cache.java

```diff
@@ -76,6 +76,7 @@ public void clear() {
         weakReferenceCache.clear();
     }
 
-
-
+    public V resourceWithCacheKeyDontRecache(K key) {
```

Comment:
This is equivalent implementation of `- (id<CBLCacheable>) resourceWithCacheKeyDontRecache: (NSString*)cacheKey;` in CBLCache.h/.m. I'd like to keep it.